lib/govspeak.rb

```diff
@@ -118,6 +132,33 @@ def preprocess(source)
       source
     end
 
+    def legislative_list_footnote_definitions(source)
+      regex = /\$LegislativeList.*?\[\^\d\]*.*?\$EndLegislativeList/m
```

Comment:
We only use the regex in the conditional on line 139 - we could probably take the same approach as you've taken for `footnotes` on the next line, and do the scanning here:

```suggestion
      is_legislative_list = source.scan(/\$LegislativeList.*?\[\^\d\]*.*?\$EndLegislativeList/m).size.positive?
```

...then use this on line 139 as `if is_legislative_list && footnotes.size.positive?`